ui-tests/cypress/support/pageObjects/pages/designer/DesignerPage.js

```diff
@@ -88,6 +90,14 @@ export default class DesignerPage extends Content {
     },
   };
 
+  initWindowOpenChecker() {
```

Comment:
What is this needed for?

Reply:
this was supposed to be used for scenario test `select a content and a content template that is unrelated or inconsistent with the content type, then try to update the Content widget configuration. Publish the page and click on Preview/View published page` (it's being commented out for now due to security restriction). it will check anything that invokes `window.open` to open the given url into the same window instead of `_blank`.

Reply:
got it; move it to the cypress commands file then, as it is not an operation specific for this page

Reply:
done